src/ansys/simai/core/data/training_data.py

```diff
@@ -164,21 +164,16 @@ def upload_part(
     def upload_folder(
         self,
         folder_path: Path,
-        compute: bool = True,
     ) -> List["TrainingDataPart"]:
         """Upload all the parts contained in a folder to a :class:`~ansys.simai.core.data.training_data.TrainingData` instance.
 
-        This method automatically requests computation of the training data
```

Comment:
One could leave bits of this sentence, WDYT? For instance, by replacing with
"Upon upload completion, SimAI will extract data from each part."

Reply:
@tmpbeing